examples/hpc-gke.yaml

```diff
@@ -30,7 +30,8 @@ deployment_groups:
     settings:
       subnetwork_name: gke-subnet
```

Comment:
Do the names on line 31 and 33 have to always same ? If yes how can we prevent user error by allowing them to specify different names ( should we add a validation to prevent this ) ?

Reply:
This doesn't always need to be the same. There can be multiple subnetworks, and each subnetwork may or may not contain secondary CIDR ranges

On preventing user error: for reference blueprints, we can do something like this to keep both in sync:
```
      subnetworks:
      - subnet_name: $(vars.deployment_name)-sub-0
        subnet_region: $(vars.region)
        subnet_ip: 192.168.0.0/18
      secondary_ranges:
      - subnetwork_name: $(vars.deployment_name)-sub-0
        ranges:
        - range_name: pods
          ip_cidr_range: 10.4.0.0/14
        - range_name: services
          ip_cidr_range: 10.0.32.0/20
```
This is also the purpose of this PR